WalletWasabi/JsonConverters/Bitcoin/CoinJsonConverter.cs.cs

```diff
@@ -0,0 +1,38 @@
+using NBitcoin;
+using Newtonsoft.Json;
+
+namespace WalletWasabi.JsonConverters.Bitcoin;
+
+public class CoinJsonConverter : JsonConverter<Coin>
```

Comment:
This file is called: `CoinJsonConverter.cs.cs`. One more `.cs` than needed.

Reply:
F***! Thanks.